sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 ## Release History
 
-### 4.1.0-beta.1 (Unreleased)
+### 4.1.0-beta.1 (2021-05-27)
```

Comment:
also please update Readme.md file once release goes on maven:
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos-spark_3-1_2-12/README.md

Reply:
ACK